README.md

```diff
@@ -41,6 +41,7 @@ Higher level constructs for RabbitMQ Bokers  | ![Experimental](https://img.shiel
   - [Monitoring RabbitMQ Brokers](#monitoring-rabbitmq-brokers)
   - [RabbitMQ Broker Integration with AWS Lambda](#rabbitmq-broker-integration-with-aws-lambda)
   - [Using Management HTTP API through `RabbitMqCustomResource`](#using-management-http-api-through-rabbitmqcustomresource)
+  - [Additional RabbitMQ Broker Examples](#additional-rabbitmq-broker-examples)
```

Comment:
Please change that to _External Examples_.

Reply:
Addressed by new changes.